### ISSUE TITLE
small poplar trees seed not to exist anymore

### ISSUE THREAD

**Issue by fluxionary** [mt-mods/moretrees#17]
all poplar saplings grow into the same size (ongen included). i'm pretty sure it wasn't always this way.

also, the icon for the small poplar ongen sapling is incorrect.

**Comment by Niklp09** [comment]
confirmed

**Comment by S-S-X** [comment]
Yeah, it seemed to be planned change but I didn't like it.
100% support for restoring small poplars and also keeping old challenges for making those small poplars actually grow.
At least it was different in many ways which actually did make it very special.

**Comment by wsor4035** [comment]
assuming s/seed/sapling from the title, it looks like when placed the small varient is getting converted to a large one by an abm/lbm perhaps. as for the icon, it is only incorrect for the fast growth/on gen one

**Comment by wsor4035** [comment]
some weirdness happening here https://github.com/mt-mods/moretrees/blob/master/node_defs.lua#L567-L590 if i comment out the register_node, it breaks the non fast growth, and removes the texture from the fast growth, but it the small fast growth does grow properly. have to try and figure out what is up with this workaround code

**Comment by wsor4035** [comment]
ok, will be pushing a fix to master shortly for this. issue caused by hacky workarounds

**Comment by wsor4035** [comment]
tldr the issue was caused by a hacky workaround of including the small variant in the same exception for the default trees, so i removed it from there, and properly supported in the regular registration while removing the old hacky workarounds
